modules/data.land/R/extract_veg.R

```diff
@@ -70,7 +71,9 @@ extract_veg <- function(new_site, start_date, end_date,
    }else{
       code_col    <- "species_USDA_symbol"
       format_name <- "usda"
-      obs[obs$species_USDA_symbol != "2PLANT", ] #removes the rows with 2PLANT, this is a NEON specific code that means they could not identify the species 
+      obs <- obs[obs$species_USDA_symbol != "2PLANT" &  
+                   obs$species_USDA_symbol != "2PLANT-H" &
+                   obs$species_USDA_symbol != "CATO6", ] #removes the rows with 2PLANT, this is a NEON specific code that means they could not identify the species 
```

Comment:
Why would you remove matches to CATO6? That's not a NEON-specific unmatched code, but a specific Hickory species that's fairly common and perfectly valid to include.

Reply:
solved